.vscode/settings.json

```diff
@@ -12,6 +12,7 @@
 
         "**/plugins/*/*/behaviors/*/partials/*.htm": "php",
         "**/plugins/*/*/components/*/*.htm": "wintercms-twig",
+        "**/plugins/*/*/components/partials/*/*.htm": "wintercms-twig",
```

Comment:
Can the above line be changed to `"**/plugins/*/*/components/**/*.htm": "wintercms-twig",` and work the same?

Reply:
Yes. So it's more correct.